src/monitor.js

```diff
@@ -212,10 +235,22 @@ async function monitor(options) {
   }
 
   if (opts.patchGlobal) {
-    process.on('uncaughtException', (err) => {
+    process.on('uncaughtException', async (err) => {
+      console.log('Uncaught Exception! Attempting to report to Sentry and crash.');
       console.log(err.stack);
```

Comment:
I would also like: `console.log("error properties: %j", err)` To get a dictionary of properties on the error object.